packages/docs/community/release-process/base-cli-contractkit-dappkit-utils.md

```diff
@@ -0,0 +1,81 @@
+# Release Process for Base, Celocli, ContractKit, DAppKit, and Utils
+
+## Versioning
+
+Use the standard MAJOR.MINOR.PATCH semantic versioning scheme described at [semver.org](http://semver.org).
+
+New releases can be expected as follows:
```

Comment:
there were no modifications made to the versioning scheme i linked (semver.org), which is why I didn't add that However i can copy/paste it here if you think it would provide clarity

Reply:
I meant more like, if changes to the ABI or storage layer would affect major, minor, patch releases but the cli API itself didn't change. but if that doesn't apply, then this is fine.